File: ch08/src/main/java/com/example/ch08/IndexController.java
package com.example.ch08;

import org.springframework.web.bind.annotation.RequestMapping;

/**
 * 类名：IndexController
 * 描述：TODO
 * 时间：2018/2/7 18:00
 *
 * @author <a href="2455676683@qq.com">zzq</a>
 */
//@Controller
public class IndexController {
    @RequestMapping("/")
    public String index() {
        return "index";
    }

}

File: ch12/src/main/kotlin/com/example/ch12/IndexController.kt
package com.example.ch12

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

/**
 * 类名：IndexController
 * 描述：TODO
 * 时间：2018/2/10 0:53
 * @author <a href="2455676683@qq.com">zzq</a>
 */
@RestController
class IndexController {
    @GetMapping("/hello")
    fun index(): String = "Hello World"
}